cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -301,22 +295,25 @@ func (sub EthereumSub) getAllClaims(fromBlock int64, toBlock int64) []types.Ethe
 	log.Printf("Replay get all ethereum bridge claim from block %d to block %d\n", fromBlock, toBlock)
 
 	var claimArray []types.EthereumBridgeClaim
-	client, err := tmClient.New(sub.TmProvider, "/websocket")
+	c, err := tmClient.New(sub.TmProvider, "/websocket")
 	if err != nil {
 		log.Printf("failed to initialize a client, error is %s\n", err.Error())
 		return claimArray
 	}
 
-	if err := client.Start(); err != nil {
+	if err := c.Start(); err != nil {
 		log.Printf("failed to start a client, error is %s\n", err.Error())
 		return claimArray
 	}
 
-	defer client.Stop() //nolint:errcheck
+	defer c.Stop() //nolint:errcheck
 
 	for blockNumber := fromBlock; blockNumber < toBlock; {
 		tmpBlockNumber := blockNumber
-		block, err := client.BlockResults(&tmpBlockNumber)
+
+		ctx := context.Background()
```

Comment:
Shouldn't we be passing a context into this function?

Reply:
This looks like an issue that existed outside of the upgrade, should create a separate ticket to create a root context and pass it around.